scripts/deployment/goerli-testnets.json

```diff
@@ -0,0 +1,61 @@
+{
+  "L1": {
+    "name": "Goerli",
+    "rpc": "https://eth-goerli.g.alchemy.com/v2/lk5Vo1HAubyYMairTVMtKHj0ynpulnBb",
```

Comment:
Are we ok with storing our Alchemy endpoints in public like this? @fredo ?

Reply:
Sorry, forgot my alchemy endpoints here. Will swap them with the public rpc endpoints.

Reply:
✅